modules/gapi/src/api/gcomputation.cpp

```diff
@@ -165,6 +165,42 @@ void cv::GComputation::apply(const std::vector<cv::Mat> &ins,
     apply(std::move(call_ins), std::move(call_outs), std::move(args));
 }
 
+cv::GRunArgs cv::GComputation::apply(GRunArgs &&ins, GCompileArgs &&args)
```

Comment:
I'd add a comment here that this version is for Python mainly

Reply:
Added